website/frontend/views/plugins.py

```diff
@@ -3,17 +3,19 @@
 import json
 from urllib import urlopen
 from flask import current_app, Blueprint, render_template
-
+from website.frontend.views.api import plugins_json_file
 plugins_bp = Blueprint('plugins', __name__)
 
 
 @plugins_bp.route('/')
 def show_plugins():
-    ordered_plugins = OrderedDict()
-    plugins_json_file = os.path.join(
-        current_app.config['PLUGINS_REPOSITORY'], "plugins.json")
-    with open(plugins_json_file, "r") as fp:
-        plugins = json.loads(fp.read().decode("utf-8"))['plugins']
-        for key in sorted(plugins, key=lambda k: plugins[k]['name'].lower()):
-            ordered_plugins[key] = plugins[key]
-    return render_template('plugins.html', plugins=ordered_plugins)
+    all_plugins = OrderedDict()
+    for version, build_version in sorted(current_app.config['PLUGIN_VERSIONS'].items(), reverse=True):
```

Comment:
hmm, how would you like to implement it? I tried to keep the dictionary ordered in config, but apparently flask sorts it in ascending order when we access it :\